lib/src/modules/task.cc

```diff
@@ -485,11 +485,11 @@ ActionResponse Task::callAction(const ActionRequest& request)
         // Pass input on stdin ($input)
         task_input = task_execution_params.get<lth_jc::JsonContainer>("input").toString();
     } else {
-        if (task_input_method.empty() || task_input_method == "stdin") {
```

Comment:
I think this `task_input_method.empty()` is implementing the `both` default. It might be better to actually _set_ `task_input_method` to `both` if it's empty (I think somewhere around here we already default it to `powershell` in some cases, so there's a pattern for it) and then not have to check `empty()` anymore.

Reply:
wow, that is a much better solution. thanks